src/lib/storage/fake_device_factory.rb

```diff
@@ -212,9 +212,9 @@ def set_topology_attributes!(disk, args)
         io_size = args["io_size"]
         min_grain = args["min_grain"]
         align_ofs = args["align_ofs"]
-        disk.topology.optimal_io_size = io_size.size if io_size && io_size.size > 0
+        disk.topology.optimal_io_size = io_size.size if io_size && !io_size.empty?
```

Comment:
"empty for a size" ? And what does "size for size" mean as in the original?

Anyway, if something has size greater than zero than it's not empty... 
For me it's better readable.